keps/sig-storage/20191125-bucket-provisioning.md

```diff
@@ -492,6 +483,34 @@ These are the common steps to delete a `Bucket`. Note: there are atypical workfl
 + If the release policy is "Retain" then the `Bucket` remains "Released" and it can potentially be reused.
 + When the COSI controller sees the `Bucket`'s `conditions[x].Reason is "Deleted", it deletes all the finalizers and the real deletions occur.
 
+###  Setting Access Permissions
+#### Dynamic Provisioning
+Incoming `BucketAccessRequest`s either contains a *serviceAccountName* where a cloud provider supports identity integration, or an *accessSecretName*. In both cases, the incoming `BucketAccessRequest` represents a request for a user to access the `Bucket`.
+When requesting access to a bucket, workloads will go through the following  scenarios described here:
++  New User: In this scenario, we do not have user account in the backend storage system as well as no access for this user to the `Bucket`. 
+	+ Create user account in the backend storage system.
+	+ add the access privileges for the user to the `Bucket`.
+	+ return the credentials to the workload owning the `BucketAccessRequest`.
++  Existing User and New Bucket: In this scenario, we have the user account created in the backend storage system, but the account is not associated to the `Bucket`.
+	+ add the access privileges to the `Bucket`.
+	+ return the credentials to the workload owning the `BucketAccessRequest`.
++  Existing User and existing Bucket: In this scenario, the user account has access policy defined on the `Bucket`.  The existing user privileges in the backend may conflict with the privileges that the user is requesting.
+	+ FAIL, if existing access policy is different from the requested policy.
+	+ if the existing privileges match the requested privileges, return the credentials to the workload owning the `BucketAccessRequest`.
++ A Service Account specified and the cloud platform identity integration maps Kubernetes ServiceAccounts to the account in the backend storage system. No need to create credentials here.
+	
+Upon success, the `BucketAccess` instance is ready for the app workload to consume.
```

Comment:
The app workload isn't really _consuming_ the BA. It is consuming the endpoint and secret volume/files added by the cosi-node-apapter.

Reply:
simply say "Upon success, the `BucketAccess` instance is ready and the app workload can access backend storage." ?